basecoin/src/helper.rs

```diff
@@ -0,0 +1,58 @@
+use std::fmt::Debug;
+
+use basecoin_app::BaseCoinApp;
+use basecoin_modules::auth::{AuthAccountKeeper, AuthAccountReader};
+use basecoin_modules::bank::Bank;
+use basecoin_modules::context::{prefix, Identifiable};
+use basecoin_modules::ibc::Ibc;
+use basecoin_store::context::ProvableStore;
+use basecoin_store::impls::RevertibleStore;
+use basecoin_store::utils::SharedRwExt;
+
+/// Gives access to the IBC module.
+pub fn ibc<S>(app: BaseCoinApp<S>) -> Ibc<RevertibleStore<S>>
```

Comment:
we never used `ibc` and `bank` functions. can we remove this file ?

Reply:
In downstream projects, these serve as convenient methods callable on an instance of `BaseCoinApp<S>`, providing access to either ibc or bank modules. Let's retain these and observe how changes would appear in the `ibc-testkit`. If unnecessary, let's remove them at that time.